config/reports.yaml

```diff
@@ -16,7 +16,7 @@ ingress:
       hosts:
         - reports.jenkins.io
         - reports.jenkins-ci.org
```

Comment:
Note: there is no associated record in the `jenkins-ci.org` DNS zone.

To be removed in a later PR?